tao_compiler/mlir/disc/transforms/lower_tf.cc

```diff
@@ -100,7 +100,9 @@ class ConvertSqueezeOpDynamic : public OpRewritePattern<TF::SqueezeOp> {
       }
       llvm::SetVector<int64_t> squeeze_dims;
       for (int64_t i = 0; i < squeeze_dims_attr_size; ++i) {
-        squeeze_dims.insert(squeeze_dims_attr[i].cast<IntegerAttr>().getInt());
+        int64_t dim = squeeze_dims_attr[i].cast<IntegerAttr>().getInt();
+        if (dim == -1) dim += input_ty.getRank();
```

Comment:
From https://www.tensorflow.org/api_docs/python/tf/squeeze#args, dim in `[-rank, rank)` are all valid.

Reply:
Done. I also add a UT for `-rank` case.